config/api.yml

```diff
@@ -103,10 +103,43 @@
     :identifier: chargeback
     :options:
     - :collection
-    :methods: *70174834086080
+    :methods: *70174834084700
     :klass: ChargebackRate
     :subcollections:
     - :rates
+    :collection_actions:
+      :post:
+      - :name: create
+        :identifier: chargeback_rates_new
+      - :name: edit
+        :identifier: chargeback_rates_edit
+      - :name: delete
+        :identifier: chargeback_rates_delete
+    :resource_actions:
+      :post:
+      - :name: edit
+        :identifier: chargeback_rates_edit
+      - :name: delete
+        :identifier: chargeback_rates_delete
+      :delete:
+      - :name: delete
+        :identifier: chargeback_rates_delete
+  :currencies:
+    :description: Currencies
+    :identifier: currency
```

Comment:
Are there plans to add this and the measure identifier below to the product features?